doc/whats_new.rst

```diff
@@ -230,6 +230,9 @@ Enhancements
      (`#6846 <https://github.com/scikit-learn/scikit-learn/pull/6846>`_)
      By `Sebastian Säger`_ and `YenChen Lin`_.
 
+   - Added new return type ``(data, target)`` : tuple option to :func:`load_iris` dataset. (`#7049 <https://github.com/scikit-learn/scikit-learn/pull/7049>`_) 
```

Comment:
I can fix this in the next PR (adding ```return_X_y``` option to load_breast_cancer dataset)
Also, the position of the ```versionadded``` tag. What do you think ?